packages/components/src/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -46,6 +46,7 @@ export default function Breadcrumbs({ labels, match }) {
 }
 
 const breadcrumbLabels = {
+  '/about': 'About',
```

Comment:
the Breadcrumbs component is not currently used and its purpose / behaviour will be changing so no need to include changes here for now

Reply:
Removed from breadcrumbs